public/app/plugins/panel/geomap/utils/regularShapes.ts

```diff
@@ -0,0 +1,98 @@
+import { Fill, RegularShape, Stroke, Style } from 'ol/style';
+
+const square = (color: string) =>
```

Comment:
should we make a function like:
```
type markerCreator( color: string, fillColor: string, radius: number ) => ol.Style
```

And implement all markers with that?

Reply:
is color for borderColor? so it is clearer than just color.